ide/html.indexing/nbproject/org-netbeans-modules-html-indexing.sig

```diff
@@ -0,0 +1,42 @@
+#Signature file v4.1
+#Version 1.0
+
+CLSS public java.lang.Object
+cons public init()
+meth protected java.lang.Object clone() throws java.lang.CloneNotSupportedException
+meth protected void finalize() throws java.lang.Throwable
+meth public boolean equals(java.lang.Object)
+meth public final java.lang.Class<?> getClass()
+meth public final void notify()
+meth public final void notifyAll()
+meth public final void wait() throws java.lang.InterruptedException
+meth public final void wait(long) throws java.lang.InterruptedException
+meth public final void wait(long,int) throws java.lang.InterruptedException
+meth public int hashCode()
+meth public java.lang.String toString()
+
+CLSS public org.netbeans.modules.html.editor.api.index.HtmlIndex
+fld public final static int VERSION = 2
+fld public final static java.lang.String NAME = "html"
+fld public final static java.lang.String REFERS_KEY = "imports"
+innr public static AllDependenciesMaps
+meth public java.util.Collection<org.openide.filesystems.FileObject> find(java.lang.String,java.lang.String)
+meth public java.util.List<java.net.URL> getAllRemoteDependencies() throws java.io.IOException
+meth public org.netbeans.modules.html.editor.api.index.HtmlIndex$AllDependenciesMaps getAllDependencies() throws java.io.IOException
+meth public org.netbeans.modules.web.common.api.DependenciesGraph getDependencies(org.openide.filesystems.FileObject)
+meth public static org.netbeans.modules.html.editor.api.index.HtmlIndex get(org.netbeans.api.project.Project) throws java.io.IOException
+meth public static org.netbeans.modules.html.editor.api.index.HtmlIndex get(org.netbeans.api.project.Project,boolean) throws java.io.IOException
+meth public void addChangeListener(javax.swing.event.ChangeListener)
+meth public void notifyChange()
+meth public void removeChangeListener(javax.swing.event.ChangeListener)
+supr java.lang.Object
+hfds INDEXES,changeSupport,querySupport
+
+CLSS public static org.netbeans.modules.html.editor.api.index.HtmlIndex$AllDependenciesMaps
```

Comment:
Uh-oh, no.

Reply:
Actually - this class IS used, and is returned from the API. It's used a long time from html.refactoring, should I somehow narrow the API somehow (seems to be used just partially).

Reply:
If it is used, then it is OK.

Reply:
I have a prepared PR to make the class final + constructor nonpublic, but I think it would be formally against Friend API rules: there's no time for potential contributors to depending modules to comment; the `HtmlIndex` class has moved and had this public exposure for several releases. I propose to incompatibly-change this Friend API in the `master`.